vdr/didsubject/management.go

```diff
@@ -55,28 +54,15 @@ type MethodManager interface {
 }
 
 // DocumentManager is the interface that groups several higher level methods to create and update DID documents.
+// Deprecated
 // Only used for V1 API calls.
 type DocumentManager interface {
```

Comment:
Why are Update and RemoveVerificationMethod still relevant in this interface? I see a deprecation warning on Update, but not on the Remove function?